lib/dateTime.js

```diff
@@ -620,6 +620,7 @@ export function getDateTimeDescriptor() {
 			dateFormats[1] = 'MMM d yyyy';
 			dateFormats[2] = 'yyyy-MM-dd';
 			dateFormats[4] = 'MMMM d';
+			dateFormats[5] = 'MMM d';
```

Comment:
The CLDR doesn't have a special entry here for `fr-CA`, but I'm good either way.